### PR TITLE
C2C-319: Add Question: Is the child fully vaccinated?

### PR DESCRIPTION
**Pull request by kazlaw**
Link to ticket [here](https://mekomsolutions.atlassian.net/browse/C2C-319)

### CODE REVIEW

**Comment by kazlaw** [comment]
[Screencast from 29-04-24 20:27:06.webm](https://github.com/mekomsolutions/openmrs-config-c2c/assets/36139599/7be7cee6-7053-4bcc-9ae3-2efe0e32731f)

File: configuration/bahmniforms/vaccinations.json [hunk 1]
```diff
@@ -1,109 +1,153 @@
...
+      "uuid": "f2812522-ad57-48f3-a33d-4e4b38283eb1",
+      "name": "Vaccinations",
+      "version": "1",
+      "published": true,
+      "auditInfo": {
+          "creator": {
+              "uuid": "A4F30A1B-5EB9-11DF-A648-37A07F9C90FB",
+              "display": "daemon",
+              "links": [
+                  {
```
Comment (Ruhanga) [inline configuration/bahmniforms/vaccinations.json:13]: Let's leave such fields untouched. You only want to copy over the `"resources":` section with the necessary translations, not forgetting to update the relevant reference UUIDs.

**Review by Ruhanga — CHANGES_REQUESTED** [review]

File: configuration/bahmniforms/vaccinations.json (hunk 1 above)
Comment (kazlaw) [reply configuration/bahmniforms/vaccinations.json]: This has been addressed [here](https://github.com/mekomsolutions/openmrs-config-c2c/pull/75/commits/1af73243f15bfc91c3c9b70df80861628913b0cf)

[Screencast from 08-05-24 14:48:31.webm](https://github.com/mekomsolutions/openmrs-config-c2c/assets/36139599/6df515d5-43b4-4a3e-96fb-29101b40a01a)

Cc: @Ruhanga

File: configuration/bahmniforms/vaccinations.json [hunk 2]
```diff
@@ -1,109 +1,153 @@
...
+              "uuid": "f274b3df-2154-11ec-8de1-0242ac1b0003",
+              "display": "superman",
+              "links": [
+                  {
+                      "rel": "self",
+                      "uri": "http://openmrs:8080/openmrs/ws/rest/v1/user/f274b3df-2154-11ec-8de1-0242ac1b0003"
+                  }
+              ]
+          },
+          "dateChanged": "2021-09-30T04:41:12.000+0000"
```
Comment (Ruhanga) [inline configuration/bahmniforms/vaccinations.json:30]: @kazlaw, this shouldn't be apart of the relevant changes in this PR. Could you undo these?

Comment (kazlaw) [reply configuration/bahmniforms/vaccinations.json:30]: @Ruhanga those changes are a reverting back of some of the changes the Implementer .json file. They were intended,

**Review by Ruhanga — APPROVED** [review]

**Pull request merged**